File: Selenium/Activities/Activity4.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity4 {

	public static void main(String[] args) {
		WebDriver driver= new ChromeDriver();
		driver.get("https://v1.training-support.net/selenium/target-practice");
		System.out.println(driver.getTitle());
		
		String HeaderText=driver.findElement(By.xpath("//h3[@id='third-header']")).getText();
		System.out.println("Third heading text is: "+HeaderText);
		
		String HeaderTextColor= driver.findElement(By.xpath("//h5[text()='Fifth header']")).getCssValue("color");
		System.out.println("Fifth Header text color is: "+HeaderTextColor);
		
		String voiletButtonClass= driver.findElement(By.xpath("//button[text()='Violet']")).getAttribute("class");
		System.out.println("Violet button classes are: "+voiletButtonClass);
		
		String GreyButtonText= driver.findElement(By.xpath("//button[text()='Grey']")).getText();
		System.out.println("Grey button text is: "+GreyButtonText);
		
		driver.quit();
		

	}

}

File: Selenium/Activities/activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/target-practice")
    print("Page title is: ", driver.title)

    HeaderText = driver.find_element(By.XPATH, "//h3[@id='third-header']")
    print("Third heading text is: ", HeaderText.text)

    HeaderTextColor = driver.find_element(By.XPATH, "//h5")
    print("Fifth heading colour is: ", HeaderTextColor.value_of_css_property("color"))

    VioletButtonClass = driver.find_element(By.CLASS_NAME, "violet")
    print("Violet button classes are: ", VioletButtonClass.get_attribute("class"))

    GreyButtonText = driver.find_element(By.CLASS_NAME, "grey")
    print("grey button text: ", GreyButtonText.text)

    driver.close()